crosswalks/csdms.csv

```diff
@@ -0,0 +1,74 @@
+Property,csdms
+codeRepository,Source web address
+programmingLanguage,Programming language / Other Programming language
+review,
+runtimePlatform,
+targetProduct,
+applicationCategory,module_type
+applicationSubCategory,
+downloadUrl,
+fileSize,
+installUrl,
+memoryRequirements,Memory requirements
+operatingSystem,Supported platforms / Other platform
+permissions,
+processorRequirements,Code optimized for (single/multiple processors)
```

Comment:
Not a good match, "processorRequirements" is defined as " Processor architecture required to run the application (e.g. IA64). "